README.md

```diff
@@ -8,27 +8,23 @@
 ```sh
 $ npm install -g standard-changelog
 $ cd my-project
-$ standard-changelog CHANGELOG.md -w
+$ standard-changelog
 ```
 
-This will *not* overwrite any previous changelog. The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".
+The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".
 
-If you first time use this tool and want to generate all previous changelog, you could do
+If you first time use this tool and want to generate all previous changelog, you can run:
 
 ```sh
-$ standard-changelog -i CHANGELOG.md -w -r 0
+$ standard-changelog -r 0
 ```
 
-This *will* overwrite any previous changelog if exist.
-
 All available command line parameters can be listed using [CLI](#cli) : `standard-changelog --help`.
 
-**Hint:** You can alias your command or add it to your package.json. EG: `"changelog": "standard-changelog -i CHANGELOG.md -w -r 0"`.
-
+**Hint:** You can alias your command or add it to your package.json. EG: `"changelog": "standard-changelog"`.
```

Comment:
If we're going to mention the use of an npm run script, I think we should be clearer about how this affects installation (`npm i --save-dev` vs `npm i -g`) and where in _package.json_ the snippet belongs (under `"scripts"`).

I think we should take an approach similar to here: https://github.com/nexdrew/conventional-recommended-workflow/tree/readme-suggestions#installation

Perhaps we can make more README improvements in a subsequent PR?